src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -29,16 +29,17 @@
           >
             <KIcon
               class="selected-item-icon"
-              :color="item.selected ? 'var(--blue-200)' : 'var(--grey-400)'"
+              :class="{ 'is-item-selected': item.selected }"
+              :color="item.selected ? 'currentColor' : `var(--grey-400, var(--kui-color-text-neutral-weak, ${KUI_COLOR_TEXT_NEUTRAL_WEAK}))`"
```

Comment:
question: why the switch to `currentColor` here? Which color does it get? I feel like readability suffers a bit with the usage

Reply:
Switched to `currentColor` as a workaround to use one of the `$tmp-` variables for color. See `is-item-selected` class definition. We don't have a token for color I needed to use here so I needed to create a `$tmp-` variable - but because we can't use SCSS variables in the template (they won't compile) - I needed a way to assign color in style tag.